ci-operator/step-registry/cucushift/installer/rehearse/aws/upi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-aws-upi-ovn-sdn2ovn-provision-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-aws-upi-ovn-sdn2ovn-provision
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install
```

Comment:
Looks installer team is still working on the aws-upi base profile, we should wait until they complete the base profile. I think the steps above are for IPI install not UPI.

Reply:
@lihongan @jianlinliu Seems aws-upi base profile is still not available yet: https://github.com/openshift/release/tree/master/ci-operator/step-registry/cucushift/installer/rehearse/aw

Reply:
Yes, upi aws PR is pending on dev's review.